packages/domains-table/src/domains-table/domains-table-row.tsx

```diff
@@ -129,160 +136,191 @@ export function DomainsTableRow( { domain }: DomainsTableRowProps ) {
 	const handleDomainLinkClick = ( e: MouseEvent ) =>
 		isAllDomainManagementEnabled ? undefined : e.stopPropagation();
 
-	return (
-		<tr
-			key={ domain.domain }
-			className={ clsx( 'domains-table__row', {
-				'is-selected': currentlySelectedDomainName === domain.domain,
-			} ) }
-			onClick={ domainManagementLink ? handleSelect : undefined }
-		>
-			{ canSelectAnyDomains && (
-				// eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-noninteractive-element-interactions
-				<td
-					className="domains-table-checkbox-td"
-					onClick={ ( e: MouseEvent ) => e.stopPropagation() }
-				>
-					<CheckboxControl
-						__nextHasNoMarginBottom
-						checked={ isSelected }
-						onChange={ () => handleSelectDomain( domain ) }
-						/* translators: Label for a checkbox control that selects a domain name.*/
-						aria-label={ sprintf( __( 'Tick box for %(domain)s', __i18n_text_domain__ ), {
-							domain: domain.domain,
-						} ) }
-						disabled={ ! canBulkUpdate( domain ) }
-					/>
-				</td>
-			) }
-
-			{ domainsTableColumns.map( ( column ) => {
-				if ( column.name === 'domain' ) {
-					return (
-						// The in-view ref is attached to the domain cell because the <tr> is display:contents, which appears to break the in-view logic
-						<td
-							key={ domain.domain + column.name }
-							className="domains-table-row__domain"
-							ref={ ref }
-						>
-							{ shouldDisplayPrimaryDomainLabel && <PrimaryDomainLabel /> }
-							{ domainManagementLink ? (
-								<a
-									className="domains-table__domain-name"
-									href={ domainManagementLink }
-									onClick={ handleDomainLinkClick }
-								>
-									{ domain.domain }
-								</a>
-							) : (
-								<span className="domains-table__domain-name">{ domain.domain }</span>
-							) }
-
-							{ isCompact && <div>{ renderSiteCell() }</div> }
+	const handlePointToWpcom = async () => {
+		setIsLoadingButton( true );
+		try {
+			await wpcomRequest( {
+				path: '/domains/point-to-wpcom',
+				apiNamespace: 'wpcom/v2',
+				method: 'POST',
+				body: {
+					domain: domain.domain,
+				},
+			} );
+			dispatch( successNotice( __( 'Domain pointed to WordPress.com' ) ) );
```

Comment:
```suggestion
			dispatch( successNotice( __( 'Domain is now pointing to WordPress.com' ) ) );
```
Maybe this reads more naturally?